app/src/main/res/values/strings.xml

```diff
@@ -558,6 +558,12 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <!-- Quests -->
     <!-- sorted alphabetically by title -->
 
+    <string name="quest_accepts_cards">Can you pay here with card?</string>
```

Comment:
“With card” sounds awkward to me. The only word that refers to both is “plastic”, but that’s slang. These are slightly longer but read more naturally:

> Can you pay by credit or debit card here?
> Are credit or debit cards accepted here?

Reply:
changed to Are credit or debit cards accepted here?